runner.go

```diff
@@ -129,6 +135,14 @@ func (r *Runner) Start() {
 			return
 		}
 
+		// Warn the user if they are watching too many dependencies.
+		if r.watcher.Size() > saneViewLimit {
```

Comment:
Maybe move this before the `once` check so that the output still shows up for that case?